### PR TITLE
BC001-PS01-Autodesk Civil3D

### PR DESCRIPTION
**Pull request by jmirtschin**
Submission of PS01 export from Autodesk Civil 3D

### CODE REVIEW

**Comment by evandroAlfieri** [comment]
Thanks @jmirtschin. Before merging (so that the railway team can start their review), can you provide some context to the team regarding the json file? (what it is, what it does, why it's been submitted, etc.)

**Comment by jmirtschin** [comment]
Hi @evandroAlfieri 
I will suppress this file if you determine that is preferred outcome.  Here is my reasoning for including it.
The test cases help improve confidence for at least 3 groups, buildingSMART, implementers and users.

I included the DWG file as it can help users (and others) replicate/review the test case (more so in the more advanced test cases than this one in particular).  The IFC export in Civil 3D can be configured or controlled by the json file (the exporter will recognize the file adjacent to the dwg).  Users might be interested in the settings used, and if the files are to be regenerated in the future (say IFC4.4 or IFC5) then having the configuration saved with the dwg could be beneficial in that event.

Let me know if the reasoning is not clear.

File: tests/PS01/test-output/Autodesk Civil 3D/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This folder contains the IFC files produced by Autodesk Civil 3D software for test case PS01.
+It contains a Civil 3D DWG file with coordinate system assigned and configuration file used for export.
```
Comment (evandroAlfieri) [inline tests/PS01/test-output/Autodesk Civil 3D/README.md:2]:
```suggestion
It contains a Civil 3D DWG file with coordinate system assigned and configuration file (.json) used for export.

Note: the IFC export in Civil 3D can be configured or controlled by the json file (the exporter will recognize the file adjacent to the dwg).
```

**Review by evandroAlfieri — APPROVED** [review]

**Comment by apoce-boo** [comment]
Report available here [https://app.box.com/s/ciju0xwvpqf6pzothscnlagcuvf9n2ji](https://app.box.com/s/ciju0xwvpqf6pzothscnlagcuvf9n2ji)

**Pull request merged**